### ISSUE TITLE
Is there a way to set a username and email for each domain?

### ISSUE THREAD

**Issue by shelllee** [ForkIssues/TrackerWin#1368]
![image](https://user-images.githubusercontent.com/11704124/141452401-21e348ea-0042-4440-8945-3f2f7c44a6f5.png)

Is there a way to set a username and email for each domain, such as `github.com`, `gitlab.com` and `company.com`?

**Comment by DanPristupov** [comment]
No. Remote server is not related to the credentials you use on commit. A repository may have multiple remote servers or none.

FYI: https://stackoverflow.com/a/36296990

**Comment by shelllee** [comment]
> No. Remote server is not related to the credentials you use on commit. A repository may have multiple remote servers or none.
> 
> FYI: https://stackoverflow.com/a/36296990

So, does fork support this?

**Comment by DanPristupov** [comment]
If you configure global git config that way, conditional credentials will also be used when you commit in Fork.

**Comment by shelllee** [comment]
perhaps a `post-checkout` hook(https://github.com/DrVanScott/git-clone-init) is more suitable for me.
